EldenRingFPSUnlockAndMore/MainWindow.xaml

```diff
@@ -12,7 +12,7 @@
         <DockPanel>
             <StackPanel DockPanel.Dock="Top" Margin="10,10,10,0" Width="300" Height="Auto">
                 <DockPanel LastChildFill="False">
-                    <CheckBox x:Name="cbFramelock" DockPanel.Dock="Left" Margin="0,0,0,0" Height="25" FontSize="14 px" VerticalContentAlignment="Center" Content="Frame rate lock:" ToolTip="Make sure to read the guide" IsChecked="{Binding Source={x:Static p:Settings.Default}, Path=FrameLockTicked, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}" TabIndex="1" />
+                    <CheckBox x:Name="cbFramelock" DockPanel.Dock="Left" Margin="0,0,0,0" Height="25" FontSize="14 px" VerticalContentAlignment="Center" Content="Frame rate lock:" ToolTip="Make sure to read the guide" IsChecked="{Binding Source={x:Static p:Settings.Default}, Path=FrameLockTicked, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}" TabIndex="1" IsEnabled="False" />
```

Comment:
Bug is right here... FrameTick won't be enabled anymore.

Reply:
yeah, the enabling of the checkbox when the game was launched was removed, so I already removed the IsEnabled="false" on my branch